src/components/FeatureViews/Registration/RegistrationLanding.js

```diff
@@ -187,31 +187,33 @@ const RegistrationLanding = () => {
                     </Tabs>
                 </Grid>
             </Grid>
-            <Grid
-                container
-                layout="row"
-                spacing={8}>
+            {view !== 1 ?
```

Comment:
To make this only render when `view !== 1`, instead of using a ternary with an empty alternate value, you can simply [use the `&&` operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) - if the condition is false, the whole thing is replaced with `false`, and React will not render `false`, `null`, or `undefined`:

    view !== 1 && <>your JSX here</>